test/extended/util/framework.go

```diff
@@ -2066,6 +2066,11 @@ func DoesApiResourceExist(config *rest.Config, apiResourceName string) (bool, er
 		return false, err
 	}
 	for _, apiResourceList := range allResourceList {
+		gvns := strings.Split(groupVersionName, "/")
+		gvs := strings.Split(apiResourceList.GroupVersion, "/")
```

Comment:
Worth turning `strings.Split(groupVersionName, "/")` into a function and invoking it when comparing. E.g.
```go
func groupName(string) {
	return strings.Split(groupVersionName, "/")[0]
}
...
if groupName(groupVersionName) != groupVersionName(apiResourceList.GroupVersion) {
	continue
}
```
The function can be later extended with checks. Also, gvns probably stands for "group version name" with s as plural. Though, after splitting the gvn string the version is no longer present.

Reply:
Done